### ISSUE TITLE
Annotations are barely usable.

### ISSUE THREAD

**Issue by maandree** [mate-desktop/atril#176]
I'm running Xmonad, and moving the annotation bubble or even writting in it is seriously problematic.

**Comment by PureTryOut** [comment]
I have to agree here. Coming from Evince (but want to get rid of those awful GtkHeaderBar's), I couldn't even find how to add annotations at first. Once added, you can't remove them. And the editing popup is obnoxiously big.
